edx/edx-enterprise
# Models and admin integration

**Description:** This PR adds models and admin integration to `enterprise` app.

**JIRA:** [ENT-33](https://openedx.atlassian.net/browse/ENT-33) and [ENT-21](https://openedx.atlassian.net/browse/ENT-21)

**Dependencies:** Cross-dependent on https://github.com/edx/edx-platform/pull/13723

**Sandbox:**
* LMS: http://pr13723.sandbox.opencraft.hosting/
* Studio: http://studio-pr13723.sandbox.opencraft.hosting/

**Testing instructions:**
1. This PR contains only Django admin integration so far - smoke testing Django admin using Enterprise Customer should be enough.
2. One extension to default Django admin features is ability to export list of Enterprise Customers to CSV.
2.1. Open "Enterprise Customer" page in Django admin
2.2. Select all/some Enterprise Customers.
2.3. In "Actions" dropdown select "Export to CSV", than press "Go" button. _Expected result:_ selected Enterprise Customers are exported to CSV

**Reviewers:**
- [ ] @edx/ecommerce
